faunadb/deprecated.py

```diff
@@ -0,0 +1,21 @@
+import warnings
+import functools
+
+def deprecated(reason):
+  def decorator(old_func):
+
+    @functools.wraps(old_func)
+    def new_func(*args, **kvargs):
+      fmt = '{name}: {reason}'
```

Comment:
I believe the style we are using for the Python driver is double quotes everywhere.